lib/elixir/lib/application.ex

```diff
@@ -145,6 +145,19 @@ defmodule Application do
   """
   @callback stop(state) :: term
 
+  @doc """
+  Called after an application starts.
+
+  This function is called after an application starts and before any included
```

Comment:
The included applications may or may not be started, it is up the application. It could be that an included application is started as a supervisor child, or is started more dynamically by a start phase. I think this phrasing is also confusing because the start phases are called for each phase on all applications (should included applications have the phase). It would be nice to show this in the example below.